src/components/nodes/String.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="q-px-md q-pt-md">
+    <q-input
+      filled
+      autogrow
+      v-model="text"
+      type="textarea"
+      placeholder="Enter text here"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  spec: {
+    id: 'std::text',
+    title: 'Text',
+    icon: require('../../assets/node_icons/node_text.svg'),
+    inputs: [],
+    outputs: [
+      {
+        id: 'value',
+        name: 'Value',
+        type: 'text'
+      }
+    ]
+  },
+  data: () => ({
+    text: ''
+  })
+}
+</script>
```

Comment:
Please move this back to `Text.vue` and instead import it under a different name. See my comment on import prefixing in `Canvas.vue`.